contributing/code/bc.rst

```diff
@@ -32,6 +32,14 @@ The second section, "Working on Symfony Code", is targeted at Symfony
 contributors. This section lists detailed rules that every contributor needs to
 follow to ensure smooth upgrades for our users.
 
+.. warning::
+
+    Code marked with the ``@experimental`` or the ``@internal`` tags are
+    excluded from our Backwards Compatibility promise.
+
+    Also note that backward compatibility breaks are tolerated if they help fix
```

Comment:
The "s" misses here too (I think it is actually correct, but it seems to be written with an "s" [in the bc promise doc](https://symfony.com/doc/current/contributing/code/bc.html))